File: Proxy/src/main/java/com/example/Proxy/dto/EventoDto.java
package com.example.Proxy.dto;
import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.time.Instant;
import java.util.List;
@Data
@AllArgsConstructor
@NoArgsConstructor
public class EventoDto {

    private String titulo;
    private String resumen;
    private String descripcion;
    private Instant fecha;
    private String direccion;
    private String imagen;
    private Integer filaAsientos;
    private Integer columnAsientos;
    private Double precioEntrada;
    private EventoTipoDTO eventoTipo;
    private List<IntegrantedDto> integrantes;
    private Long id;



}

File: Frontend/composeApp/src/commonMain/kotlin/org/example/project/dto/EventoDTO.kt
package org.example.project.dto
import kotlinx.serialization.Serializable

@Serializable
data class EventoDTO(
    val id: Long,
    val titulo: String,
    val fecha: String,
    val descripcion: String,
    val precioEntrada: Double,
    val eventoTipo: EventoTipoDTO,
)

@Serializable
data class EventoTipoDTO(
    val nombre: String,
    val descripcion: String
)